openfe/protocols/openmm_utils/settings_validation.py

```diff
@@ -71,3 +71,95 @@ def get_simsteps(sim_length: unit.Quantity,
         raise ValueError(errmsg)
 
     return sim_steps
+
+
+def convert_steps_per_iteration(
```

Comment:
Doesn't this behaviour differ from how get_simsteps behave? I would prefer that we align our divisions/conversions. Especially worth considering the whole attosecond division stuff.

Reply:
Going over the MD protocol - I'm probably missing something really obvious but isn't this method exactly the same as using `get_simsteps` but with an MC value of 1 (like we do there)?